### DISCUSSION TITLE
Hyprland error

### DISCUSSION THREAD

**Discussion by sgj123456** [bevyengine/bevy#9279] [Q&A] [answered]
ERROR sctk_adwaita::config: XDG Settings Portal did not return response in time: timeout: 100ms, key: color-scheme

**Comment by nicopap** [comment]
@sgj123456 could you say how you fixed it? It's good for people who might get similar errors in the future. https://xkcd.com/979/

**Reply by sgj123456** [reply]
Sorry, I don't know. After several updates, there was no such issue anymore

**Comment by JstRafau** [comment, marked as answer]
Even though this discussion is closed, it's still unanswered, so I decided to comment.
I managed to fix it, either by changing default icon theme from Adwaita to breeze-dark (or presumably any other that isn't Adwaita) in `/usr/share/icons/default/index.theme`,
or with a `gsettings set org.gnome.desktop.interface color-scheme prefer-dark` command that I found [on reddit](https://www.reddit.com/r/gnome/comments/rzknul/how_do_i_switch_the_freedesktop_colorscheme/).
I do believe that both ways work because that error disappeared either way, but because of some other unrelated error in sctk-adwaita I reverted my changes to the default icon theme, and then some time later I used `gsettings` to finally fix it.
